manifest.json

```diff
@@ -602,6 +602,16 @@
       "useCaseName": "Multi-step workflow"
     }
   ],
+  "rust(provided.al2)": [
```

Comment:
Can you the runtime name here to `rust (provided.al2)` as against `rust(provided.al2)`.

Reply:
Done!